Seneca-CDOT/telescope
# Remove unnecessary graphs in dashboard

## Issue This PR Addresses



## Type of Change




## Description


This PR removes all the unused graphs from dashboard. It is needed to cleanup the 
dashboard.

## Steps to test the PR


1. `pnpm dev` to start the `status` service
2. Go to `localhost:1111` 
3. No graphs should be shown

## Checklist
